### PR TITLE
Create AccountActivityRestClient

### PR DESCRIPTION
**Pull request by julien-pons**
Add Rest Client to handle Twitter Premium AccountActivity API
https://developer.twitter.com/en/docs/accounts-and-users/subscribe-account-activity/api-reference/aaa-premium

### CODE REVIEW

File: src/main/scala/com/danielasfregola/twitter4s/entities/Webhook.scala [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package com.danielasfregola.twitter4s.entities
+
+final case class Webhook(id: String, url: String, valid: Boolean, created_timestamp: String) // TODO - convert to Instant? parsers doesn't support actual format
```
Comment (DanielaSfregola) [inline src/main/scala/com/danielasfregola/twitter4s/entities/Webhook.scala:3]: About this TODO: you can write a custom parser to support whatever format they use.
We should probably be doing that before releasing this PR...if you do not have the time/do not know how to do it, I can have a look in a couple of weeks :)

File: src/test/resources/fixtures/rest/accountactivity/webhook.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "created_timestamp": "2020-06-22 15:19:10 +0000",
```
Comment (DanielaSfregola) [inline src/test/resources/fixtures/rest/accountactivity/webhook.json:2]: Yeah, I think we can easily add a parser for this :)
Although probably it should be an `Instant`, but rather a `ZonedDateTime` or similar..?

File: src/main/scala/com/danielasfregola/twitter4s/entities/Webhook.scala (hunk 1 above)
Comment (julien-pons) [reply src/main/scala/com/danielasfregola/twitter4s/entities/Webhook.scala]: Yes you're right, I added a formatter to do so 🙂

File: src/test/resources/fixtures/rest/accountactivity/webhook.json (hunk 2 above)
Comment (julien-pons) [reply src/test/resources/fixtures/rest/accountactivity/webhook.json:2]: Indeed, I used `ZonedDateTime`

File: src/main/scala/com/danielasfregola/twitter4s/http/serializers/CustomFormats.scala [hunk 3]
```diff
@@ -40,6 +40,15 @@ private[twitter4s] case object LocalDateSerializer
         case date: LocalDate => JString(date.toString)
       }))
 
+private[twitter4s] case object ZonedDateTimeSerializer
```
Comment (DanielaSfregola) [inline src/main/scala/com/danielasfregola/twitter4s/http/serializers/CustomFormats.scala:43]: Well done! :D

Comment (julien-pons) [reply src/main/scala/com/danielasfregola/twitter4s/http/serializers/CustomFormats.scala:43]: Thank's :D 
And by the way, do you know when you will create a new release that includes these changes ?

Comment (DanielaSfregola) [reply src/main/scala/com/danielasfregola/twitter4s/http/serializers/CustomFormats.scala:43]: I'll do my best to release within 2 weeks :)

Comment (julien-pons) [reply src/main/scala/com/danielasfregola/twitter4s/http/serializers/CustomFormats.scala:43]: Nice, that sounds great !
Thanks for your time :)

Comment (DanielaSfregola) [reply src/main/scala/com/danielasfregola/twitter4s/http/serializers/CustomFormats.scala:43]: version 7.0 contains this change and it is on its way to maven central :)

Comment (julien-pons) [reply src/main/scala/com/danielasfregola/twitter4s/http/serializers/CustomFormats.scala:43]: Wonderful 🎉
Thank's a lot !

**Pull request merged**